cmake-modules/FindCheck.cmake

```diff
@@ -0,0 +1,60 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
What is this file for?

Reply:
Could you help me with it? I used it inspired by cmake articles, but I'm not sure which license we need to adopt (I'm a little bit newbie with licensing, so they still confused for me :sweat_smile: ). Please feel free to change it to accomplish Orcania's licensing.